cmd/buildah/main.go

```diff
@@ -61,12 +61,6 @@ func main() {
 			Usage: "storage driver option",
 			Value: defaultStoreDriverOptions,
 		},
-		cli.StringFlag{
```

Comment:
Since we are using lipbod/pkg/secrets in buildah, the code doesn't support reading this path of the default mounts file from this. It just uses the actual override and default mount file paths, so there was no point in having this as we can't test with it.
I could modify libpod/pkg/secrets for this one case.